region/lowernorfair/east.json

```diff
@@ -4181,6 +4396,12 @@
                           "hits": 3
                         }]}
                       ]},
+                      {"and": [
```

Comment:
I think I remember asking about that and you said the setup was too unreliable. Was it somewhere else? Did you find a good setup?

Reply:
That might have been about the Kihunter shaft, I don't quite remember. This thing was already in the reqs but it was wrongly inside the kihunter obstacle.